File: PYTHON VERSION/inUvectors.py
# inUvectors
"""
Created on Thu Sep 28 15:18:09 2023

@author: simone.lucertini
"""

import Unit_struct as Us


U1 = dict(Us.Unit)
U1["Bindex"] = 0
U1["ulocal"]  = [ 1.0, 0]

U2 = dict(Us.Unit)
U2["Bindex"]= 1
U2["ulocal"]= [ 1.0, 0]

Uvectors = [U1, U2]
del (U1, U2)

File: _references/Nikravesh - Planar Multibody Dynamics 2nd Edition/DAP_JC_12_2018/DAP_JC 12_2018/Models/SP/inUvectors.m
function inUvectors
    include_global

V1 = Unit_struct;
V1.Bindex = 0;
V1.ulocal  = [1; 0];

V2 = Unit_struct;
V2.Bindex = 1;
V2.ulocal  = [1; 0];

Uvectors = [V1; V2];
